lms/static/js/views/image_field.js

```diff
@@ -208,9 +209,9 @@
             onBeforeUnload: function () {
                 var status = this.getCurrentStatus();
                 if (status === 'uploading') {
-                    return gettext("Upload is in progress. To avoid errors, stay on this page until the process is complete.");
+                    return gettext('Upload is in progress. To avoid errors, stay on this page until the process is complete.');  // jshint ignore:line
```

Comment:
How does this actually get used?  It needs to be escaped where it is used, but I'm not clear where that would be.